lib/node_modules/@stdlib/stats/base/dists/binomial/cdf/src/main.c

```diff
@@ -0,0 +1,59 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include "stdlib/stats/base/dists/binomial/cdf.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/factorial.h"
+#include "stdlib/math/base/special/pow.h"
+
+/**
+* Evaluates the cumulative distribution function (CDF) for a binomial distribution with number of trials `n` and success probability `p` at a value `k`.
+*
+* @param k    input value
+* @param n    number of trials
+* @param p    success probability
+* @return     evaluated CDF
+*
+* @example
+* double y = stdlib_base_dists_binomial_cdf( 5, 10, 0.5 );
+* // returns ~0.623
+*/
+double stdlib_base_dists_binomial_cdf( const int k, const int n, const double p ) {
```

Comment:
```suggestion
double stdlib_base_dists_binomial_cdf( const int64_t k, const int64_t n, const double p ) {
```

Use fixed width types. Also means you need to include `stdint.h` above.